testdata/bad_directive.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 The Cockroach Authors.
```

Comment:
nit: I don't think we want this license header (we should also remove the one we have in `noescape.go`).

Reply:
Fixed